### PR TITLE
Venkatesh dev

### PR DESCRIPTION
**Pull request by venki17393**
The post URL now serves the request dynamically and the corresponding JSON is displayed to user

### CODE REVIEW

File: src/com/session/SessionHelper.java [hunk 1]
```diff
@@ -46,37 +48,65 @@ public static boolean isPresent(String email) {
...
-		while ((line = br.readLine()) != null) {
-			sb.append(line + "\n");
+		ObjectMapper mapper = new ObjectMapper();
+		PostPojo post = null;
+		JSONObject json = new JSONObject();
+		try {
+			String temp = requestNumber.substring(1);
+			int lastChar = Integer.parseInt(temp);
+			System.out.println(lastChar);
+			if (lastChar > 0 && lastChar < 11) {
```
Comment (ramesh-lingappan) [inline src/com/session/SessionHelper.java:70]: what if lastChat is null ?

File: src/com/session/SessionHelper.java [hunk 2]
```diff
@@ -46,37 +48,65 @@ public static boolean isPresent(String email) {
...
+				while ((line = br.readLine()) != null) {
+					sb.append(line + "\n");
+				}
+				br.close();
+
+				json.put("ok", true);
+				json.put("message", "found");
+				String string = sb.toString();
+				String jsonString = json.toString();
+				string=string.concat(jsonString);
```
Comment (ramesh-lingappan) [inline src/com/session/SessionHelper.java]: may i know what is happening here ?

File: src/com/session/SessionHelper.java [hunk 3]
```diff
@@ -46,37 +48,65 @@ public static boolean isPresent(String email) {
...
+				post = mapper.readValue(string, PostPojo.class);
+				System.out.println(string);
+				//post = mapper.readValue(jsonString, PostPojo.class);
+
+				return post;
+			} else {
+
+				json.put("ok", false);
```
Comment (ramesh-lingappan) [inline src/com/session/SessionHelper.java]: this method is meant to return only Post Pojo, not other api response properties like ok, message etc, those needs to be done in servlets , based on pojo is null or not

File: src/com/session/PostPojo.java [hunk 4]
```diff
@@ -6,13 +6,27 @@
 	private int id;
 	private String title ;
 	private String body ;
+	private String ok;
```
Comment (ramesh-lingappan) [inline src/com/session/PostPojo.java]: this property should not be in pojo

File: src/com/session/PostPojo.java [hunk 5]
```diff
@@ -6,13 +6,27 @@
...
+	private String message;
```
Comment (ramesh-lingappan) [inline src/com/session/PostPojo.java]: this property should not be in pojo

**Review by ramesh-lingappan — CHANGES_REQUESTED** [review]

**Comment by venki17393** [comment]
Yea it is causing an exception i will edit and again make the pull request

**Pull request closed without merging**